### PR TITLE
Added the Nod missile silo and the Ion Cannon to the Tiberian Sun mod

### PR DESCRIPTION
**Pull request by Mailaender**
This adds http://cnc.wikia.com/wiki/Missile_Silo_(Tiberian_Sun) and http://cnc.wikia.com/wiki/Multi_missile

### CODE REVIEW

**Comment by abcdefg30** [comment]
Needs a rebase.

**Comment by Mailaender** [comment]
Rebased.

**Comment by abcdefg30** [comment]
What about
```
	Selectable:
		Bounds: 75,48
```
for ```NAMISL```

and for ```GAPLUG```
```
	Selectable:
		Bounds: 115,120,0,-20
```
?

Otherwise :+1: .

**Comment by Mailaender** [comment]
Thanks. Updated.

File: mods/ts/rules/structures.yaml [hunk 1]
```diff
@@ -1288,6 +1288,108 @@ NAPULS:
 	Power:
 		Amount: -150
 
+NAMISL:
+	Inherits: ^Building
+	Buildable:
+		Queue: Defense
+		BuildPaletteOrder: 100
+		Owner: nod
```
Comment (abcdefg30) [inline mods/ts/rules/structures.yaml]: Please remove this.

**Comment by huwpascoe** [comment]
:+1:

**Pull request merged**